src/freenet/node/useralerts/UserAlert.java

```diff
@@ -24,6 +25,13 @@ public interface UserAlert {
 	 */
 	public String getText();
 
+	/**
+	 * Content of the body of the alert (no metadata).
+	 */
+	default public String getMessageText() {
+		return getText();
+	}
```

Comment:
Uhm… that’s a _third_ method now that returns the text of the user alert. Why do we need a _third_ method for this? All (one) places this is used from have a `N2NTMUserAlert` instance which implements the `UserAlertFromPeer` interface, giving us access to this method. There is no reason for further polluting `UserAlert`.